src/niobot/client.py

```diff
@@ -769,9 +769,27 @@ async def create_dm_room(
         result = await self.room_create(
             is_direct=True,
             invite=[user_id],
+            visibility=nio.api.RoomVisibility.private,
+            preset=nio.api.RoomPreset.trusted_private_chat,
         )
         if isinstance(result, nio.RoomCreateError):
             raise GenericMatrixError("Failed to create DM room", response=result)
+        
+        dm_rooms = {}
+        dm_rooms[user_id] = [result.room_id]
```

Comment:
Just being pedantic here but wouldn't it be cleaner to just do `dm_rooms = {user_id: [result.room_id]}`?